assemble/bin/start-daemon.sh

```diff
@@ -103,6 +103,9 @@ if [[ "$SERVICE" != "tserver" || $NUM_TSERVERS -eq 1 ]]; then
    fi
    echo "Starting $SERVICE on $HOST"
 
+   OUTFILE="${ACCUMULO_LOG_DIR}/${SERVICE}_1_${LOGHOST}.out"
```

Comment:
The if section is used when there is only one tserver and for all the services (master, gc, etc).